lib/util/stream.js

```diff
@@ -98,7 +98,10 @@ class PartialPipe {
     }
   }
   with(stream) { return new PartialPipe(this.streams.concat([ stream ])); }
-  pipeline(errCb) { return pipeline(this.streams, rejectIfError(errCb)); }
+  pipeline(errHandler) {
+    this.streams[this.streams.length-2].on('error', errHandler);
+    return pipeline(this.streams, () => {});
```

Comment:
Maybe this change is specific to `streamErrorHandler()`, and should only be implemented in `lib/http/endpoint.js`.

Reply:
Teesting at https://app.circleci.com/pipelines/github/alxndrsn/odk-central-backend/228/workflows/8564a967-914b-4e96-8b00-f6a338fc043d/jobs/240